### PR TITLE
task: enable perfsprint

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

Enable perfsprint so we stop doing `fmt.Errorf` when we can do `errors.New`

### CODE REVIEW

File: internal/cli/kubernetes/config/apply.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func ApplyBuilder() *cobra.Command {
 }
 
 func getOperatorMajorVersion(image string) string {
-	version := strings.Trim(image, fmt.Sprintf("%s:", containerImage))
+	version := strings.TrimPrefix(image, containerImage+":")
```
Comment (gssbzn) [inline internal/cli/kubernetes/config/apply.go:176]: @mongodb/atlas_kubernetes_team this was an interesting one, removing fmt flagged an incorrect usage of Trim and that we should use TrimPrefix, can you confirm?

Comment (s-urbaniak) [reply internal/cli/kubernetes/config/apply.go:176]: technically this behaves differently but indeed the `TrimPrefix` makes much more sense. LGTM from my side but leaving the review to have another set of eyes to @josvazg @helderjs @roothorp

**Review by helderjs — APPROVED** [review]
LGTM

File: internal/cli/kubernetes/config/apply.go (hunk 1 above)
Comment (josvazg) [reply internal/cli/kubernetes/config/apply.go:176]: yeah LGTM as well, we are only interesting on removing a prefix, that should not happen also at the end.

**Pull request merged**